run_dir/design/base.html

```diff
@@ -58,24 +58,17 @@
                       <li><a href="/applications">Application Counts</a></li>
                       <li><a href="/production/cronjobs">Cronjobs List</a></li>
                       <li><a href="/flowcells_plot">Flowcells Trends</a></li>
+                      <li><a href="/data_delivered_plot">Data Delivered Trends</a></li>
                       <li role="separator" class="divider"></li>
                       <li class="dropdown-header">Tools</li>
                       <li><a href="/nas_quotas">NAS Quotas</a></li>
                       <li><a href="/reads_total/">Read Count Totals</a></li>
                       <li><a href="/proj_meta">Compare Project Meta</a></li>
                       <li><a href="/instrument_logs">View Instrument Logs</a></li>
                       <li><a href="/barcode">Barcode and Label printing</a></li>
-                      <li><a href="/pricing_quote">Cost Calculator</a></li>
-                      {% if user and (user.is_admin or user.is_pricing_admin) %}
-                        <li role="separator" class="divider"></li>
-                        <li class="dropdown-header">Admin</li>
-                        {% if user and user.is_admin %}
-                        <li><a href="/user_management">User Management</a></li>
-                        {% end %}
-                        {% if user and user.is_pricing_admin %}
-                        <li><a href="/pricing_preview">Edit Cost Calculator</a></li>
-                        {% end %}
-                      {% end %}
+                      {% if user and user.is_admin %}
+                      <li><a href="/user_management">User Management</a></li>
+                      {% end %}</li>
```

Comment:
Hmm, I think something has gone wrong here with some merge. You will overwrite my changes from the pricing if this PR is merged currently... All the pricing_admin things in the base.html would be lost.

Reply:
Right, forgot about this!